src/style.scss

```diff
@@ -32,6 +32,7 @@
 .xwp-country-card-flag {
 	font-size: calc(200px + 3vw);
 	line-height: 1;
+	z-index: 20;
```

Comment:
NIT: I think that this works only because `z-index` implicitly changes an element `position` from `static` to `relative`. So I guess `position: relative` instead of `z-index: 20` would do here, too.